db/seeds.rb

```diff
@@ -81,7 +81,7 @@ def dictionary_entries
 def transit_lines
   TransitLine.destroy_all
   ActiveRecord::Base.connection.reset_pk_sequence!("transit_lines")
-  CSV.foreach('db/fixtures/transit_lines.csv', :headers => true) do |csv_obj|
+  CSV.foreach('db/fixtures/transit_lines.csv', :headers => true,encoding: utf-8 ) do |csv_obj|
```

Comment:
Line is too long. [94/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator '-'.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.